### ISSUE TITLE
Scenario outline with examples in title

### ISSUE THREAD

**Issue by marcfallows** [cucumber/cucumber-js#267]
Hello,

I would expect to be able to have my scenario outline title have more information by using the examples within the title itself:

```
Scenario Outline: A <some> step is <result>
   When a <some> step
   Then I get <result>
Examples:
   | some    | result  |
   | passing | passed  |
   | failing | skipped |
```

Then my scenario titles end up very useful:
`Scenario: A passing step is passed`
`Scenario: A failing step is skipped`

This doesn't seem to be working for me. Is this a supported feature? If not, is it something that I could assist in having included? Or, is it not expected to work like this?

**Comment by aslakhellesoy** [comment]
I just realised [Cucumber-JVM already does this](https://groups.google.com/d/msg/cukes/3G_2tmSBwh4/USOiNMIF5hgJ). I can't remember if I implemented that myself or if it was someone else. Thanks for the heads up @brasmusson !

**Comment by chrismilleruk** [comment]
Thanks @aslakhellesoy,

I've sent a PR across, let me know if you need any alterations.

**Comment by sejtan88** [comment]
This feature will help me very much. Any ETA when this will be released? I have copied source code and it is working fine, but it would be good to have it in npm install.

**Comment by chrismilleruk** [comment]
Hi, just a gentle nudge to @jbpros, @mattwynne and @aslakhellesoy on this.

It would be great if we could get some feedback on the PR.

Thanks
Chris

**Comment by mattwynne** [comment]
If this behaviour is already in Cucumber-JVM, and people want it here, I'm not going to stand it it's way. Merge on!

Don't expect me to rush to implement it in Ruby though :grinning:

**Comment by jbpros** [comment]
@chrismilleruk can you rebase off master please?

**Comment by chrismilleruk** [comment]
@jbpros all done.

I've left it as two commits since it's easier to diff that way. Let me know if you'd like me to squash it.

**Comment by chrismilleruk** [comment]
Hi @jbpros, 

Just checking in to see if there's anything I can do to assist further here.

**Comment by riaan53** [comment]
Awesome!

Can we get a merge for PR: #291 please :)

Thank you @chrismilleruk and the rest!

**Comment by jbpros** [comment]
Closed by 4673a59.

**Comment by JimTheMan** [comment]
I'm with @mattwyne. Scenario Outlines are great. They allow you to separate the business logic of the test from the individual equivalence partition values, and it's easier for people who didn't write the scenarios to edit the data values later on.
